frontend/src/model.tsx

```diff
@@ -956,10 +958,23 @@ async function GetAppConfig() {
   }
 }
 
-async function LoadInitialState() {
+async function LoadCsrfToken(logger: ILogger): Promise<string> {
+  try {
+    const token = await getCsrfToken(axiosInstance)
+    setCsrfTokenHeader(axiosInstance, token)
+    logger.info(`Set X-CSRF-Token header to ${token}`)
+    return token
+  } catch (error) {
+    logger.error('Could not fetch CSRF token')
+    throw error
+  }
+}
+
+async function LoadInitialState(logger: ILogger) {
   const region = getState(['app', 'selectedRegion'])
   clearState(['app', 'aws'])
   clearAllState()
+  await LoadCsrfToken(logger)
```

Comment:
question: failing this call will prevent everything that comes later to load up. Have you considered using at least a `notify` call to inform the user something bad has happened?

Reply:
An error in fetching the CSRF token may be hard to handle (not much we can do aside from retrying indefinitely)

However, subsequent calls would fail with a specific error code. I'll add an handler to `notify` the user and log the error when the CSRF check fails

Reply:
Coming back to this. The backend is going to return 403 Forbidden (which is already handled), I'll add an error message to the `LoadCsrfToken` command